<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tt3::tools::restore::ConfigureRestoreDialog</class>
 <widget class="QDialog" name="tt3::tools::restore::ConfigureRestoreDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Restore workspace</string>
  </property>
  <property name="windowIcon">
   <iconset resource="tt3-tools-restore.qrc">
    <normaloff>:/tt3-tools-restore/Resources/Images/Misc/RestoreLarge.png</normaloff>:/tt3-tools-restore/Resources/Images/Misc/RestoreLarge.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="QPushButton" name="restoreFromPushButton">
     <property name="text">
      <string>Browse</string>
     </property>
     <property name="icon">
      <iconset resource="tt3-tools-restore.qrc">
       <normaloff>:/tt3-tools-restore/Resources/Images/Actions/BrowseSmall.png</normaloff>:/tt3-tools-restore/Resources/Images/Actions/BrowseSmall.png</iconset>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="4">
    <widget class="QLabel" name="restoreIntoLabel">
     <property name="text">
      <string>--- Restore into ---</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="workspaceTypeLabel">
     <property name="text">
      <string>Workspace type:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="2">
    <widget class="QComboBox" name="workspaceTypeComboBox"/>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QLabel" name="locationLabel">
     <property name="text">
      <string>Location:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="locationLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="browsePushButton">
     <property name="text">
      <string>Browse</string>
     </property>
     <property name="icon">
      <iconset resource="tt3-tools-restore.qrc">
       <normaloff>:/tt3-tools-restore/Resources/Images/Actions/BrowseSmall.png</normaloff>:/tt3-tools-restore/Resources/Images/Actions/BrowseSmall.png</iconset>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="restoreFromLineEdit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>0</height>
      </size>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="restoreFromLabel">
     <property name="text">
      <string>Restore from:</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>restoreFromLineEdit</tabstop>
  <tabstop>restoreFromPushButton</tabstop>
  <tabstop>workspaceTypeComboBox</tabstop>
  <tabstop>locationLineEdit</tabstop>
  <tabstop>browsePushButton</tabstop>
 </tabstops>
 <resources>
  <include location="tt3-tools-restore.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>tt3::tools::restore::ConfigureRestoreDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>tt3::tools::restore::ConfigureRestoreDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>restoreFromPushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::tools::restore::ConfigureRestoreDialog</receiver>
   <slot>_restoreFromPushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>24</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>workspaceTypeComboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>tt3::tools::restore::ConfigureRestoreDialog</receiver>
   <slot>_workspaceTypeComboBoxCurrentIndexChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browsePushButton</sender>
   <signal>clicked()</signal>
   <receiver>tt3::tools::restore::ConfigureRestoreDialog</receiver>
   <slot>_browsePushButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>301</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>178</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>_restoreFromPushButtonClicked()</slot>
  <slot>_workspaceTypeComboBoxCurrentIndexChanged(int)</slot>
  <slot>_browsePushButtonClicked()</slot>
 </slots>
</ui>
